spec/presenters/edition_presenter_spec.rb

```diff
@@ -36,6 +36,10 @@
     it "mixes in the specified update_type to the presentation" do
       expect(subject[:update_type]).to eq update_type
     end
+
+    it "adds the supertypes" do
+      expect(subject["user_journey_document_supertype"]).to eq "thing"
```

Comment:
Pages on GOV.UK are either "finding" or "thing". It's determined by document type in:

https://github.com/alphagov/govuk_document_types/blob/master/data/supertypes.yml#L30-L53